### PR TITLE
Use count based stabilization for perf_analyzer

### PR DESCRIPTION
**Pull request by Tabrizian**
I added the Triton `format.py` file here too so that we can unify formatting across the files.

### CODE REVIEW

File: model_analyzer/perf_analyzer/perf_config.py [hunk 1]
```diff
@@ -142,8 +143,7 @@ def __getitem__(self, key):
         elif key in self._input_to_verbose:
             return self._verbose[self._input_to_verbose[key]]
         else:
-            raise TritonModelAnalyzerException(
-                f"'{key}' Key not found in config")
+            raise KeyError
```
Comment (aramesh7) [inline model_analyzer/perf_analyzer/perf_config.py:146]: Do we want to raise a key error if someone uses an improper argument to perf analyzer via `perf_analyzer_flags`?

**Review by aramesh7 — CHANGES_REQUESTED** [review]
It was a little tough to spot the relevant changes. Is there anything that can be done about this?

File: model_analyzer/perf_analyzer/perf_analyzer.py [hunk 2]
```diff
@@ -147,17 +143,34 @@ def run(self, metrics):
...
-
                 if process.returncode != 0:
                     if self._output.find(
-                            "Please use a larger time window.") > 0:
-                        self._config['measurement-interval'] += INTERVAL_DELTA
-                        logger.info(
-                            "perf_analyzer's measurement window is too small, "
-                            f"increased to {self._config['measurement-interval']} ms."
-                        )
+                            "Failed to obtain stable measurement") > 0:
```
Comment (aramesh7) [inline model_analyzer/perf_analyzer/perf_analyzer.py:148]: This section is tested by `L0_perf_analyzer` , Is there anything could you add to the tests there?

**Review by aramesh7 — CHANGES_REQUESTED** [review]

File: model_analyzer/perf_analyzer/perf_analyzer.py (hunk 2 above)
Comment (Tabrizian) [reply model_analyzer/perf_analyzer/perf_analyzer.py:148]: I'll add a unittest for it.

File: model_analyzer/perf_analyzer/perf_config.py (hunk 1 above)
Comment (Tabrizian) [reply model_analyzer/perf_analyzer/perf_config.py:146]: Good point. Will revert this change.

**Review by aramesh7 — APPROVED** [review]

**Pull request merged**